# Protocol Template

MIOP protocol template

## AUTHORS

| PREPARED BY All authors known to have contributed to the preparation of this protocol, including those who filled in the template.  | AFFILIATION | ORCID (visit https://orcid.org/ to register) | DATE |
| ------------- | ------------- | ------------- | ------------- |
| Hugo Gante | KU Leuven, the Africa Museum, Belgium  |  | yyyy-mm-dd |
| Filip Volckaert | KU Leuven, Belgium  |  | yyyy-mm-dd |
| Bart Hellemans | KU Leuven, Belgium  |  | yyyy-mm-dd |
| Enora Geslain | KU Leuven, Belgium  |  | yyyy-mm-dd |
| Saara Suominen  | IOC-UNESCO  |  | 2023-01-19 |
| Ward Appeltans  | IOC-UNESCO  |  | yyyy-mm-dd |
| Pieter Provoost  | IOC-UNESCO  |  | yyyy-mm-dd |
| Fanny Douvere  | WHC-UNESCO  |  | yyyy-mm-dd |
| Robbert Casier  | WHC-UNESCO  |  | yyyy-mm-dd |
| Masaki Miya  | Natural History Museum and Institute, Japan  |  | yyyy-mm-dd |
| Bruce Deagle  | CSIRO Australian National Fish Collection |  | yyyy-mm-dd |
| Sophie von der Heyden  | University of Stellenbosch, South Africa  |  | yyyy-mm-dd |
| Francisco Chavez  | MBARI, USA  |  | yyyy-mm-dd |
| Dianne Gleeson  |  EcoDNA, University of Canberra, Australia |  | yyyy-mm-dd |
| Craig Sherman  | Deakin University, Australia  |  | yyyy-mm-dd |
| Nicolas Pade  | EMBRC, France |  | yyyy-mm-dd |
| Kelly Goodwin  | NOAA, USA  |  | yyyy-mm-dd |
| Ole Bjorn Brodnicke  |  Copenhagen University, Denmark |  | yyyy-mm-dd |
| Naiara Rodriguez-Ezpeleta  | AZTI, Spain  |  | yyyy-mm-dd |
| Pascal Hablutzel  | VLIZ, Belgium  |  | yyyy-mm-dd |
| Julia Busch  | Common Wadden Sea Secretariat  |  | yyyy-mm-dd |
| Kim Praebl  |  The Arctic University, Norway |  | yyyy-mm-dd |
| Giomar Helena Borrero-Perez  | Marine and Coastal Research Institute, Colombia  |  | yyyy-mm-dd |
| Vanessa Yepes-Narvaez  | Marine and Coastal Research Institute, Colombia  |  | yyyy-mm-dd |

## PROTOCOL REVISION RECORD

Version numbers start at "1.0.0" when the protocol is first completed and will increase when changes that impact the outcome of the procedure are made (patches: 1.0.1; minor changes: 1.1.0; major changes: 2.0.0). Please store all versions in the gDrive folder designated to your institute.

| VERSION  | RELEASE DATE This is the date when a given protocol version was finalised | DESCRIPTION OF REVISIONS Please include a brief description of what was changed relative to the previous version |
| ------------- | ------------- | ------------- |
| 1.0.0  | yyyy-mm-dd  | Initial release  |
|   |   |   |

## RELATED PROTOCOLS IN YOUR FOLDER

This is a list of other protocols deposited in your folder which should be known to users of this protocol. For example, if you create a derivative or altered protocol, you would link to the original protocol in the section below. Please include the link to each related protocol. Also include the version number of that protocol when you linked to it.

| PROTOCOL NAME AND LINK  | VERSION The version of the protocol you linked to | RELEASE DATE This is the date corresponding to the version listed to the left |
| ------------- | ------------- | ------------- |
|   |   | yyyy-mm-dd  |
|   |   | yyyy-mm-dd  |

## RELATED EXTERNAL PROTOCOLS

This is a list of other protocols that are not in your folder which should be known to users of this protocol. These include, e.g., kit manuals. Please upload all relevant external protocols to Appendix A and link to them here.

| EXTERNAL PROTOCOL NAME AND LINK  | ISSUER / AUTHOR Please note who authored the protocol (this may also be a company name) | ACCESS DATE This is the date you downloaded or scanned the protocol and uploaded it. |
| ------------- | ------------- | ------------- |
| [UNESCO eDNA expeditions field sampling booklet](https://www.unesco.org/en/edna-training-materials)  | UNESCO  | yyyy-mm-dd  |
| UNESCO eDNA expeditions sample sheet  | UNESCO  | yyyy-mm-dd  |
| [Sylphium eDNA kit instructions](https://drive.google.com/file/d/1UNHRrUcgGFc3nfrDWDp_b0UYfTjAHi7D/view) | Sylphium   | yyyy-mm-dd  |
| [UNESCO eDNA expeditions instruction video](https://www.unesco.org/en/edna-training-materials)  | UNESCO   | yyyy-mm-dd  |
| [UNESCO eDNA expeditions sample registration app](https://app.ednaexpeditions.org/)  | UNESCO  | yyyy-mm-dd  |

## ACRONYMS AND ABBREVIATIONS

| ACRONYM / ABBREVIATION | DEFINITION |
| ------------- | ------------- |
|   |   |

## GLOSSARY

| SPECIALISED TERM | DEFINITION |
| ------------- | ------------- |
|   |   |
|   |   |

# BACKGROUND

This document describes the required protocol to conduct amplification of target genes as a part of UNESCO eDNA expeditions. 

## Summary

UNESCO Environmental DNA Expeditions is a global, citizen science initiative that will help measure marine biodiversity, and the impacts climate change might have on the distribution patterns of marine life, across UNESCO World Heritage marine sites. It is a citizen science initiative, whose objective is to engage local communities in sampling for eDNA and inspire the next generation of ocean professionals. Water sampling is based on citizen science sampling kits, which make sampling easy and with minimal required materials and training. 

This protocol explains the amplification of target biomarkers from the UNESCO eDNA expeditions project. The objectives of the project were to target fish and megavertebrates, and the benefit of citizen science engagement for this work.

## Method description and rationale

The project has a broad goal of testing eDNA sampling for the description of fish and vertebrate biodiversity at the sites sampled by citizen scientists. For this end, multiple biomarkers were chosen to ensure a large target diversity. Most markers were targeting fish, however also mammal targets were included as well as a broad-range CO1 target biomarker to give insight to biodiversity at the sites at large, compare across the vastly different sites and have the possibility of detecting invasive or other rare target species. 

Eight different biomarkers were chosen for the analysis of biodiversity from the sites. 

| Primer Set | Loci |F-primer| R-primer | Length | Reference |
|-----|-----|--------|----------|--| ---|
|Mifish-U| 12S |GTCGGTAAAACTCGTGCCAGC     |CATAGTGGGGTATCTAATCCCAGTTTG | 171 | Miya et al. 2015| 
|Mifish-E| 12S |GTTGGTAAATCTCGTGCCAGC     |CATAGTGGGGTATCTAATCCTAGTTTG | 171 | Miya et al. 2015| 
|Mimammal-U| 12S |GGGTTGGTAAATTTCGTGCCAGC     |CATAGTGGGGTATCTAATCCCAGTTTG | 171 | Ushio et al. 2017|
|Mimammal-E| 12S |GGACTGGTCAATTTCGTGCCAGC     |CATAGTGAGGTATCTAATCTCAGTTTG | 171 | Ushio et al. 2017|
|Mimammal-B| 12S |GGGTTGGTTAATTTCGTGCCAGC     |CATAGTGGGGTATCTAATCCCAGTTTG | 171 | Ushio et al. 2017|
|Teleo | 12S (L1848, H1913) |ACACCGCCCGTCACTCT    | CTTCCGGTACACTTACCATG | 63 | Valentini et al. 2016 |
|Leray| CO1 (mlCOIintF, jgHCO2198) |GGWACWGGWTGAACWGTWTAYCCYCC    | TAIACYTCIGGRTGICCRAARAAYCA | 313 | Leray et al. 2013, Geller et al. 2013 |
|Vert-16S| 16S | AGACGAGAAGACCCYdTGGAGCTT    | GATCCAACATCGAGGTCGTAA | 250 | Vences et al. 2016 |

The Mifish and Mimammal primers were utilised in one reaction, finally resulting in four separate reactions. Each reaction was done in eight replicates to increase the target DNA used in the reactions. DNA was diluted 5 times to counteract any pcr inhibition from the samples. 

## Spatial coverage and environment(s) of relevance

eDNA sampling campaigns are performed across 25 UNESCO World Heritage marine sites. All samples were treated the same for DNA extraction and PCR. 

# PERSONNEL REQUIRED

Highly trained and skilled molecular laboratory scientists/technicians are required to conduct this work. 

## Safety

Working with DNA and in the molecular laboratory requires high awareness of safety and risks associated with the chemicals. In addition, it is very important to work in a clean space, and avoiding contamination of samples at any point of the sample processing. See guidelines for clean eDNA work. 

## Training requirements

A specialized trainging (usually university level) for the molecular laboratory is required for this work.

## Time needed to execute the procedure

The time needed is highly dependant on the number of samples being processed. For up to 96 samples you should reserve about a week for this work.

# EQUIPMENT

| DESCRIPTION e.g. filter | PRODUCT NAME AND MODEL Provide the official name of the product | MANUFACTURER Provide the name of the manufacturer of the product. | QUANTITY Provide quantities necessary for one application of the standard operating procedure (e.g. number of filters). | REMARK For example, some of the consumable may need to be sterilized, some commercial solution may need to be diluted or shielded from light during the operating procedure. |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| **Durable equipment** |
| Pipettes |  |  | |  |
| Plate centrifuge|  |  | |  |
| Thermocycler | |  | |  |
| Vortex | | |  | |
| Magnetic stand |  |  |  |  |
| Tecan plate reader|  |  |  |  |
| **Consumable equipment** |
| Pipette tips|  | |  |  |
| Ice |  |  | |  |
| DNA quantification kit | Quant-iT PicoGreen dsDNA kit | Invitrogen, Life Technologies |  |  |
| PCR plate | |  |  |  |
| Aluminum foil |  |  |  |  |
| **Chemicals** |
| Enzyme for amplification | Qiagen Taq mix (2x) | Qiagen | 10 μl |  |
| Primer | Primer mix (see above) |  | 0.4 μl | 10 μM concentration |
| PCR-grade water |  |  | 4.6 μl |  |
| CleanPCR beads | CleanNA |  | 64 μl  |  |
| 70% Ethanol | |  | 200 μl  |  |
| TE-buffer | |  | 200 μl  |  |
| Enzyme for index PCR | MyTaq HS mix (2x) |  | 10 μl  |  |

# STANDARD OPERATING PROCEDURE

In the following SOP, please use the exact names of equipment as noted in the table above.

## Protocol


Preparation

All work is to be performed in a clean room specifically prepared for the processing of eDNA samples, and to avoid all possible sources of contamination.

1.  Step 1
2.  Step 2

### PCR



**1. Step 1: Master mix preparation**

- For each sample you need:
    - 10 μl Qiagen Taq mix (2x)
    - 4.6 μl H2O
    - 0.4 μl primer mix (10 μM of each primer)
- Thaw reagents and put on ice
- Prepare a master mix with these 3 components (number of samples + 10%)
- Vortex and spin down

Of the following primers, all Mifish and Mimammal forward primers and all Mifish and Mimammal reverse primers were mixed together before proceeding. 

| PCR Primer Name | Direction | Sequence (5’ -> 3’)|
| ----- | ----- | ----- |
| Mifish-U-F | forward | GTCGGTAAAACTCGTGCCAGC  |
| Mifish-U-R | reverse | CATAGTGGGGTATCTAATCCCAGTTTG |
| Mifish-E-F | forward | GTTGGTAAATCTCGTGCCAGC |
| Mifish-E-R | reverse | CATAGTGGGGTATCTAATCCTAGTTTG |
| Mimammal-U-F | forward | GGGTTGGTAAATTTCGTGCCAGC   |
| Mimammal-U-R | reverse | CATAGTGGGGTATCTAATCCCAGTTTG |
| Mimammal-E-F | forward | GGACTGGTCAATTTCGTGCCAGC  |
| Mimammal-E-R | reverse | CATAGTGAGGTATCTAATCTCAGTTTG |
| Mimammal-B-F | forward | GGGTTGGTTAATTTCGTGCCAGC  |
| Mimammal-B-R | reverse | CATAGTGGGGTATCTAATCCCAGTTTG |
| teleo_F L1848 | forward | ACACCGCCCGTCACTCT   |
| teleo_R H1913 | reverse | CTTCCGGTACACTTACCATG |
| mlCOIintF | forward | GGWACWGGWTGAACWGTWTAYCCYCC |
| jgHCO2198| reverse | TAIACYTCIGGRTGICCRAARAAYCA |
| Vert-16S-eDNA-F1 | forward | AGACGAGAAGACCCYdTGGAGCTT   |
| Vert-16S-eDNA-R1 |  reverse | GATCCAACATCGAGGTCGTAA |


**2. Step 2: PCR plate preparation**

- Aliquot 15 μl master mix into each well of the PCR plate
- Add 5 μl diluted purified gDNA to each well (include blank)
- Centrifuge shortly in plate centrifuge (max 1500 rpm).


**3. Step 3: PCR reaction on PCR cycler**

Note that there are four different reactios for the four different primer types. Each primer has a specific temperature that is used for the annealing as shown in the table below. Each reaction is performed in eight replicates.

| PCR step | Temperature | Duration | Repetition |
| ----- | ----- | ----- | ----- |
| Initial denaturation | 95°C | 15 min |  |
| Denaturaion | 95°C | 30 s | 35 times |
| Annealing | Mifish/Mimammal - 61°C <br> Teleo - 55°C <br> CO1 - 57 °C <br> 16S-vert - 65°C | 90 s | 35 times |
| Elongation | 72°C | 60 s | 35 times |
| Final elongation | 72°C | 10 min |  |
| Cooling | 10°C |  | |


Store PCR samples at 4°C for short time storage (1 day) or at -20°C for long time storage. Optional: check products on gel (limited number of samples).  

All 8 replicates were pooled and half of the volume was used for magnetic bead purification with the CleanPCR beads (CleanNA).


### Quality control, PCR clean-up

Half of the pooled PCR reaction volume was used for magnetic bead purification with the CleanPCR beads (CleanNA).
A total volume of 80 μl from each single locus PCR was used for magnetic bead purification. Final elution was done with 25 μl PCR grade water and stored at -20°C till quantification.

**1. Step 1: Mixing the magnetic beads with the DNA samples**

- Homogenize magnetic beads by vortexing
- Add 0.8x PCR volume (64 μl beads to 80 μl PCR reaction)
- Pipet up and down 10 times to ensure proper mixing
- Incubate at RT for 5 min
- Place PCR plate on magnetic stand for 5 min
- Remove the clear solution while the plate is still on the magnetic stand (5 μl less than the total volume). Avoid taking out any beads.

**2. Step 2: Washing of the beads (plate stays on magnetic stand)**

- Add 200 μl of 70% ethanol
- Incubate for 1 min (longer is no problem, DNA is safe)
- Set pipet at 230 μl and remove everything (discard in special waste container/bottle) o Add 200 μl of 70% ethanol
- Incubate for 1 min
- Set pipet at 230 μl and remove everything (discard in special waste container/bottle) 
- Remove last traces of ethanol with 10 μl pipet to doublecheck whether all wells are empty (residual ethanol may interfere with downstream analysis)

**2. Step 3: Elution of the purified DNA samples**

- Remove plate from the magnetic stand
- Add 25 μl PCR-grade water
- Pipet up and down 10 times to ensure proper mixing (avoid air bubbles at the bottom)
- Incubate at RT for 5 min
- Place PCR plate on magnetic stand for 5 min
- Prepare new PCR plate (Note: SAME ORIENTATION AS PREVIOUS ONE!)
- Transfer 22 μL to new plate (to be sure no beads are transferred, although they do not form a problem later on)

## Quality control

Concentration was measured with the Quant-iT PicoGreen dsDNA kit (Invitrogen, Life Technologies) according the manufactures guidelines.

**1. Step 1: Prepare 1 x TE buffer**
- 200 μl needed per sample
- For 100 samples prepare 20 ml 1xTE buffer
    - 1 ml 20xTE buffer
    - 19 ml high pure water
- Store in falcon tube at RT (long-term storage)

**2. Step 2: Prepare standard dilution series**
- Label 6 eppendorf tubes
- Pipet 1xTE buffer first, no need to change filter tip 
- Mix by pipetting:
    - Tube 1 = dilution (1/100; 1 ng/μl) = 198 μl 1x TE + 2 μl of Lambda stock DNA (100 ng/μl)
    - Tube 2 = dilution (1/2; 0.5 ng/μl) = 100 μl 1xTE + 100 μl tube 1
    - Tube 3 = dilution (1/2; 0.25 ng/μl) = 100 μl 1xTE + 100 μl tube 2
    - Tube 4 = dilution (1/5; 0.05 ng/μl) = 160 μl 1xTE + 40 μl tube 3
    - Tube 5 = dilution (1/5; 0.01 ng/μl) = 160 μl 1xTE + 40 μl tube 4
    - Tube 6 = Blank (1) = 100 μl 1xTE
- Pipet 100 μl of each tube into Greiner plate 

**3. Step 3: Prepare samples**
- Pipet 99 μl 1xTE buffer (prepared in step 1) into Greiner plate
- Add 1 μl DNA sample (= 1/100 dilution) 

**4. Step 4:Prepare Quant-iT reagent**
- Note: Prepare reagent just before use!
- 100μl needed per sample and standard
- For 100 samples prepare 10 ml Quant-iT reagent
    - 50 μl 200x Quant-iT reagent (blue cap)
    - 9950 μl 1xTE buffer (prepared in step 1) 
    
**5. Step 5:Add Quant-iT reagent to samples and standard dilution series**
- Add 100 μl Quant-iT reagent to each well of the Greiner plate (samples and standards) with multipipet
- Wrap plate in aluminium foil and measure within 5 – 10 minutes 
- Measure with Tecan plate reader
- Calculate concentrations

## Pooling of the different loci per sample and index PCR

As four different reactions were performed for each sample, these reactions are pooled prior to indexing. Unique indices are then added for each sample. 

**1. Step 1:Pooling and aliquoting of index primers**

- Five μl from each purified locus PCR sample was pooled before index PCR.
- Combination of 96 Unique dual index primers for each plate.

**2. Step 2:Master mix preparation**

- For each sample you need:
    - 10 μl MyTaq HS mix (2x)
    - 8 μl water
- Thaw reagents and put on ice
- Prepare a master mix with these 2 components (number of samples + 10%)
- Vortex and spin down

**3. Step 3:PCR plate preparation**

- Aliquot 18 μl master mix into each well of the PCR plate
- Add 1 μl of the unique dual index primers to each well (2.5μM each) with multi-channel pipet
- Add 1 μl pooled purified PCR’s to each well with multi-channel pipet 
-  Centrifuge shortly in plate centrifuge (max 1500 rpm).

**4. Step 4:PCR reaction on PCR cycler**

| PCR step | Temperature | Duration | Repetition |
| ----- | ----- | ----- | ----- |
| Initial denaturation | 95°C | 1 min |  |
| Denaturaion | 95°C | 15 s | 10 times |
| Annealing | 51°C | 15 s | 10 times |
| Elongation | 72°C | 10 s | 10 times |
| Final elongation | 72°C | 5 min |  |
| Cooling | 10°C |  | |

Remove plate from PCR cycler when reaction is finished!  <br><br>
Each PCR reaction is followed by a magnetic bead purification (0.8 x ratio). Final elution is done with 25 μl pure water and quantified with the picogreen kit as before.  <br><br>

**5. Step 5:Pool all samples for sequencing library**

Pool 20 ng of each sample. Add full volume of low concentrated samples

## Basic troubleshooting guide



# REFERENCES

Geller, J., Meyer, C., Parker, M., & Hawk, H. (2013). Redesign of PCR primers for mitochondrial cytochrome c oxidase subunit I for marine invertebrates and application in all-taxa biotic surveys. Molecular Ecology Resources, 13(5), 851–861. https://doi.org/10.1111/1755-0998.12138   

Leray, M., Yang, J. Y., Meyer, C. P., Mills, S. C., Agudelo, N., Ranwez, V., Boehm, J. T., & Machida, R. J. (2013). A new versatile primer settargeting a short fragment of the mitochondrial COI region for metabarcoding metazoan diversity : application for characterizing coral reef fish gut contents. Frontiers in Zoology, 10(34), 1–14. https://doi.org/10.1186/1742-9994-10-34

Miya, M., Sato, Y., Fukunaga, T., Sado, T., Poulsen, J. Y., Sato, K., Minamoto, T., Yamamoto, S., Yamanaka, H., Araki, H., Kondoh, M., & Iwasaki, W. (2015). MiFish, a set of universal PCR primers for metabarcoding environmental DNA from fishes: Detection of more than 230 subtropical marine species. Royal Society Open Science, 2(7). https://doi.org/10.1098/rsos.150088

Ushio, M., Fukuda, H., Inoue, T., Makoto, K., Kishida, O., Sato, K., ... & Miya, M. (2017). Environmental DNA enables detection of terrestrial mammals from forest pond water. Molecular Ecology Resources, 17(6), e63-e75.
https://doi.org/10.1111/1755-0998.12690

Valentini, A., Taberlet, P., Miaud, C., Civade, R., Herder, J., Thomsen, P. F., Bellemain, E., Besnard, A., Coissac, E., Boyer, F., Gaboriaud, C., Jean, P., Poulet, N., Roset, N., Copp, G. H., Geniez, P., Pont, D., Argillier, C., Baudoin, J. M., ... Dejean, T. (2016). Next-generation monitoring of aquatic biodiversity using environmental DNA metabarcoding. Molecular Ecology, 25(4), 929–942. https://doi.org/10.1111/mec.13428

Vences, M., Lyra, M. L., Perl, R. G. B., Bletz, M. C., Stanković, D., Lopes, C. M., ... Steinfartz, S. (2016). Freshwater vertebrate metabarcoding
on Illumina platforms using double-indexed primers of the mitochondrial 16S rRNA gene. Conservation Genetic Resources, 8, 323–327. https://doi.org/10.1007/s12686-016-0550-y
 

# APPENDIX A: DATASHEETS

